components/builder-api/src/http/mod.rs

```diff
@@ -46,19 +46,18 @@ pub fn router(config: Arc<Config>) -> Result<Chain> {
         authenticate: get "/authenticate/:code" => github_authenticate,
 
         jobs: post "/jobs" => XHandler::new(job_create).before(bldr.clone()),
-        job: get "/jobs/:id" => XHandler::new(job_show).before(bldr.clone()),
-        job_log: get "/jobs/:id/log" => XHandler::new(job_log).before(bldr.clone()),
+        job: get "/jobs/:id" => XHandler::new(job_show).before(basic.clone()),
+        job_log: get "/jobs/:id/log" => XHandler::new(job_log).before(basic.clone()),
```

Comment:
Looks good overall, the only thing I am wondering is if there is is risk of any sensitive information exposure in any of the logs.